docs/reference/contributing/tools/mbedls.md

```diff
@@ -1,21 +1,21 @@
-### mbed-ls
+### Mbed-ls
 
-[Add description here.]
+Mbed-ls is a Python module that detects and lists Mbed Enabled boards connected to a host computer.
```

Comment:
host computer or just a computer?

Reply:
I suppose you would like to remove "host". I'll do that.